service/src/main/kotlin/io/provenance/digitalcurrency/consortium/service/AddressTagService.kt

```diff
@@ -46,7 +47,15 @@ class AddressTagService(
             pbcService.getAttributeByTagName(addressRegistrationRecord.address, bankClientProperties.kycTagName)
         when (existing.isEmpty()) {
             true -> {
-                log.info("blockchain tag not done yet - will check next iteration.")
+                val response = pbcService.getTransaction(addressRegistrationRecord.txHash!!)
+                if (response == null || response.txResponse.isFailed()) {
```

Comment:
didnt realized failed transactions are queryable?

Reply:
it's possible. Here is an example - https://explorer.test.provenance.io/tx/3E803152920591032398B869A03FDD2AB97BBAE71EEFC21240D4772A7FA1D692